test/unittest/statistics/rtps/mock/StatisticsBase/statistics/rtps/StatisticsBase.hpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
```suggestion
// Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Reply:
I think this one should be kept as it is

Reply:
But it looks like you created the file for this PR, right?

Reply:
Sorry, completely right, I did not remember that I had to replicate it in the mocks folder